simtools/job_submission/job_manager.py

```diff
@@ -0,0 +1,207 @@
+
+        Parameters
+        ----------
+        submitCommand: str
+            job submission command
+        test: bool
+        testing mode without sub submission
+
+        """
+        self._logger = logging.getLogger(__name__)
+        self.submitCommand = submitCommand
+        self.test = test
+
+        try:
+            self.test_submission_system()
+        except MissingWorkloadManager:
+            self._logger.error(
+                "Requested workflow manager not found: {}".format(
+                    self.submitCommand))
+            raise
+
+    def test_submission_system(self):
+        """
+        Check that the requested workload manager exist on the
+        system this script is executed
+
+        Raises
+        ------
+        MissingWorkloadManager
+            if workflow manager is not found
+
+        """
+
+        if self.submitCommand is None:
+            return
+        elif self.submitCommand.find("qsub") >= 0:
+            if gen.program_is_executable('qsub'):
+                return
+            raise MissingWorkloadManager
+        elif self.submitCommand.find("condor_submit") >= 0:
+            if gen.program_is_executable('condor_submit'):
+                return
+            raise MissingWorkloadManager
+        elif self.submitCommand.find("seriell_script") >= 0:
+            return
+        elif self.submitCommand.find("more") >= 0:
+            return
+
+        raise MissingWorkloadManager
+
+    def submit(
+        self,
+        run_script=None,
+        run_out_file=None,
+    ):
+        """
+        Submit a job described by a shell script
+
+        Parameters
+        ----------
+        run_script: string
+            Shell script descring the job to be submitted
+        run_out_file: string
+            Redirect output/error/job stream to this file (out,err,job suffix)
+
+        """
+        self.run_script = str(run_script)
+        self.run_out_file = str(run_out_file)
+
+        self._logger.info(
+            'Submitting script {}'.format(self.run_script))
+        self._logger.info(
+            'Job output stream {}'.format(self.run_out_file+".out"))
+        self._logger.info(
+            'Job error stream {}'.format(self.run_out_file+".err"))
+        self._logger.info(
+            'Job log stream {}'.format(self.run_out_file+".job"))
+
+        if self.submitCommand.find("qsub") >= 0:
+            self._submit_gridengine()
+        elif self.submitCommand.find("condor_submit") >= 0:
+            self._submit_HTcondor()
+        elif self.submitCommand.find("seriell_script") >= 0:
+            self._submit_seriell_script()
+        elif self.submitCommand.find("more") >= 0:
+            self._submit_for_testing()
+
+    def _submit_for_testing(self):
+        """
+        Print submit script to screen for testing
+
+        """
+
+        self._logger.info('Testing script')
+
+        shellCommand = 'more ' + self.run_script
+
+        if not self.test:
```

Comment:
I don't understand this, if ``more`` is for testing, why does it run only when ``test`` is false? 
The use of ``more`` to show the user the submission script is good, but I would call that option ``test`` (since I can't figure out the different between ``more`` and ``test`` so far).

Reply:
See previous comment.